Source/Reporting/Web/src/app/Reporting/CaseReportsForListing/training-case-reports/training-case-reports.component.scss

```diff
@@ -48,6 +48,21 @@ th {
     }
 }
 
+
+.table>tbody>tr>td,
```

Comment:
Same comment as for live reports with regards to the selector. Can the CSS for these to tables be merged to avoid duplication?